generic/cxl.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2017 IBM
+# Author: sudeesh john <sudeesh@linux.vnet.ibm.com>
+
+import os
+
+from avocado import Test, main
+from avocado.utils import build, git, process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Cxl(Test):
+    """
+        This tests the CAPI functionality in IBM Power machines.
+        This wrapper uses the testcases from
+        https://github.com/ibm-capi/cxl-tests.git
+    """
+
+    def setUp(self):
+        """
+            Preparing the machie for the cxl test
+        """
+        self.script = self.params.get('script', default='memcpy_afu_ctx')
+        self.args = self.params.get('args', default='')
```

Comment:
some default value for `self.args` will be good

Reply:
the test has its own default values.. those will be used ..

Reply:
ah sure